tensorflow/python/ops/math_grad.py

```diff
@@ -767,6 +767,10 @@ def _SparseMatMul(t1, t2, out_dtype,
 def _FloorGrad(_, unused_grad):
   return [None]
 
+@ops.RegisterGradient("Rint")
```

Comment:
It's backward stable to always use zero as the gradient for floor, rint, ceil, etc.  I think `NoGradient` used to be the right thing to do; I don't know if someone got around renaming that to `ZeroGradient` or some such.